winlogbeat/docs/overview.asciidoc

```diff
@@ -1,14 +1,21 @@
 == Overview
 
-Winlogbeat ships Windows event logs to Elasticsearch or Logstash. It installs as
-a Windows service on all versions of Windows since XP.
+Winlogbeat ships Windows event logs to Elasticsearch or Logstash. You can
+install it as a Windows service on Windows XP or later.
```

Comment:
In order to be able to run our installation script on Windows XP the user will need to install PowerShell because it is wasn't included with XP originally. Microsoft provides an installer for XP.